xml/docker_concept.xml

```diff
@@ -0,0 +1,498 @@
+  <title>&deng; Terminology</title>
+  <remark>toms 2019-02-04: check with Tanja, if you need the full title or
+  if "Terminology" is enough.</remark>
+  <para>
+   Before going into detail about &deng;, let us define some of the terms used:
+  </para>
+  <remark>toms 2019-02-04: two things:
+   1. Does it makes sense to sort the list alphabetically? It seems, we
+      discussed that already, but maybe ask Tanja what's her opinion on that.
+   2. I've asked Tanja, and we agreed, the term should NOT be repeated
+      in the definition. For example, in the "Registry" definition, you should
+      remove the "A registry is..." and write "Storage for already created
+      images." Same for the other terms.
+  </remark>
+
+  <variablelist>
+   <varlistentry xml:id="Docker_engine">
+    <term>&deng;</term>
+    <listitem>
+     <para>
+      &deng; is a server-client type application that performs all
+      tasks related to virtual machines. &deng; comprises the
+      following:
+     </para>
+     <itemizedlist>
+      <listitem>
+       <formalpara>
+        <title>Daemon:</title>
+        <para>
+         The server side of &deng; manages all &docker; objects (images,
+         containers, network connections used by containers, etc.).
+        </para>
+       </formalpara>
+      </listitem>
+      <listitem>
+       <formalpara>
+        <title>REST API:</title>
+        <para>
+         Applications can use this API to communicate directly with the daemon.
+        </para>
+       </formalpara>
+      </listitem>
+      <listitem>
+       <formalpara>
+        <title>CLI Client:</title>
+        <para>
+         Enables you to communicate with the daemon. If the daemon is running
+         on a different machine than the CLI client, the CLI client can
+         communicate by using network sockets or the REST API provided by
+         &deng;.
+        </para>
+       </formalpara>
+      </listitem>
+     </itemizedlist>
+    </listitem>
+   </varlistentry>
+   <varlistentry xml:id="Docker_Image">
+    <term>Image</term>
+    <listitem>
+     <para>
+      An <emphasis>image</emphasis> is a read-only template<remark>toms 2019-02-04: 
+      template sounds a bit strange in this context.
+      On Quora.com, I've found an interesting definition:
+      "An image is a compressed, self-contained piece of software which has
+      to be unwrapped in order to use the functionality." Maybe you can add
+      it somehow.</remark> used to create a
+      <emphasis>virtual machine</emphasis> on the host server. A &docker; image is
+      made by a series of layers built one over the other. Each layer
+      corresponds to a permanent change, for example an update of an
+      application.
+      <remark>toms 2019-02-04: The following sentence is dangerous and
+      misleading IMHO. The term "changes" does not relate to a Dockerfile.
+      That file is used merely as a kind of "build recipe".
```

Comment:
I agree with Toms here. Maybe rephrase to something like this:
"A self-contained piece of software. A Docker image is a read-only template used to create a virtual machine on
the host server. The image is built with a Dockerfile." (I would give the explanation for Dockerfile only in the varlistentry on 'Dockerfile').

Reply:
Sorry for the commentary...let's be clear, a docker image is a file for docker. It is not a template, not a virtual machine and is unique to docker. Actually I'd avoid "docker" altogether and replace with "container" as there are multiple engines available in SLE and CaaS Platform.

Reply:
@taroth21 Changed.